tests/testthat/helper-for-tests.R

```diff
@@ -20,3 +20,7 @@ executeWithTestFile <- function(actionWithFile) {
   actionWithFile(newFile)
   file.remove(newFile)
 }
+
+comparisonTolerance <- function() {
+  1e-4
+}

```

Comment:
moved the definition of comparison tolerance here.

Reply:
Good idea.
Would writing `comparisonTolerance <- function(tol = 1e-4) {tol}` be better ?
This way we can also tune to other values if necessary.

Reply:
yes!

Reply:
done